pkg/ctl/delete/cluster.go

```diff
@@ -72,17 +70,13 @@ func doDeleteCluster(cmd *cmdutils.Cmd, force bool, disableNodegroupEviction boo
 			return err
 		}
 	}
-	cmdutils.LogRegionAndVersionInfo(meta)
 
+	cmdutils.LogRegionAndVersionInfo(meta)
 	logger.Info("deleting EKS cluster %q", meta.Name)
 	if err := printer.LogObj(logger.Debug, "cfg.json = \\\n%s\n", cfg); err != nil {
 		return err
 	}
 
-	if ok, err := ctl.CanDelete(cfg); !ok {
```

Comment:
Can delete notes:

- `CanDelete`
    - Already called NewProvider before it
    - Probably can remove the TTL
    - Interesting `CanDelete` was called only once, but even so, it just errored out if Refresh failed. But that would have happened before it anyways